tutor/commands/upgrade/common.py

```diff
@@ -41,6 +43,23 @@ def upgrade_from_nutmeg(context: click.Context, config: Config) -> None:
     )
 
 
+def get_mongo_upgrade_parameters(
+    docker_version: str, compatibility_version: str
+) -> tuple[int, dict[str, int | str]]:
+    """
+    Helper utility to get parameters required during mongo upgrade.
+    """
+    mongo_version = int(docker_version[0])
```

Comment:
Minor tweak to ensure that this function doesn't break at Mongo v10
```suggestion
    mongo_version = int(docker_version.split('.')[0])
```